examples/platform/linux/Options.cpp

```diff
@@ -94,6 +102,9 @@ enum
 #if CONFIG_BUILD_FOR_HOST_UNIT_TEST
     kDeviceOption_SubscriptionResumptionRetryIntervalSec = 0x1026,
 #endif
+#if CHIP_WITH_NLFAULTINJECTION
+    kDeviceOption_FaultInjection = 0x1027,
```

Comment:
Would have liked to see an integration test make use of the fault injection just added.

Reply:
Added to my TODO list for a follow-up: https://github.com/project-chip/connectedhomeip/issues/32541